src/links/v1/providers.json

```
@@ -144,10 +144,10 @@
       "type": "links",
       "id": "oqmd",
       "attributes": {
-        "name": "open quantum materials database",
-        "description": "",
-        "base_url": null,
-        "homepage": null,
+        "name": "The Open Quantum Materials Database",
+        "description": "The OQMD is a database of DFT calculated thermodynamic and structural properties of materials",
+        "base_url": "http://oqmd.org/optimade",
```

This would be the only change required in the providers file, you would then need to add an `oqmd/` folder that follows the same structure as e.g. https://github.com/Materials-Consortia/providers/tree/master/src/index-metadbs/mp/v1

```suggestion
        "base_url": "http://providers.optimade.org/index-metadbs/oqmd",
```

---

It looks like this PR still hasn't made it in, but the instructions are all there: https://github.com/Materials-Consortia/providers/pull/18

---

> It looks like this PR still hasn't made it in, but the instructions are all there if you wanted to self-host #18 or here https://github.com/Materials-Consortia/providers/tree/master/src/index-metadbs if you wanted to use this repo

I have just merged this now. So you can read the instructions directly in the repository README on the landing page.

---

Thanks, @ml-evs and @CasperWA !  I've added a new directory in `index-metadb` and changed the API response from HTML to JSON.